style.css

```diff
@@ -1,4 +1,4 @@
-@charset "UTF-8";
+@charset "UTF-8"; /* stylelint-disable-line */
```

Comment:
@ntwb Can Would you mind taking a look why Travis still shows 

>  2:1  ✖  Expected empty line before comment   comment-empty-line-before

in https://travis-ci.com/WordPress/twentytwenty/jobs/241941980 despite `/* stylelint-disable-line */` had been added behind `@charset "UTF-8";`? Should the statement look like this instead:

```
/* stylelint-disable-line custom-property-empty-line-before */
```

Reply:
Yes checking

Reply:
@Swapnildhanrale I think @ntwb needs to take a look here. 😉 While you could add an empty row before the statement to solve this issue, the file should not start with an empty line.